striptease/data_interface.py

```diff
@@ -84,3 +84,105 @@ def load_sci_data(fname_or_data, polarimeter, detector, data_type):
     scitime = astropy.time.Time(scidata["m_jd"], format="mjd").unix  # time SCI in sec
 
     return scitime, scidata[data_type + detector]
+
+
+###########################################################################################################
+
+
+class load_hk_data:  # (fname_or_data, polarimeter, hk_type, hk_parameter):
```

Comment:
Please follow the convention used in all the code in `striptease`: use [CamelCase](https://en.wikipedia.org/wiki/Camel_case) instead of [snake_case](https://en.wikipedia.org/wiki/Snake_case). So, define class `LoadHkData` instead of `load_hk_data`.

If we consistently adopt the same naming convention everywhere, users will not need to remember whether to use camel/snake case when remembering how to call one of Striptease's classes.

Reply:
>  I looked into the doc folder but there is little documentation regarding the code and the documentation that is present is not clear to me. It would be very useful to have an example of documentation written in the correct format of a class containing methods and attributes.

A very good start is the file `docs/tutorial.rst`. You could create a new file `docs/file_access.rst` following this template and then write some text describing how you can use the interface. Just a few paragraphs and some real examples will be enough. Be sure to list `tutorial.rst` in file `docs/index.rst`, so that it will be included in the manual when the docs are produced.

> Regarding camels and snakes I will convert to the correct format. I also noticed some inconsistencies in the striptease code regarding this convention

CamelCase is used for class names, snake_case for functions. (Sorry, I implicitly assumed this information in my comment above.) This is used consistently in the code, apart from those classes that are automatically generated by the Qt library:

```
$ grep "^class [A-Za-z]\+_[A-Za-z]\+" $(find . -name "*.py")
./program_viewer/ui/main_window.py:class Ui_MainWindow(object):
./postmortem_viewer/ui/__init__.py:class Ui_mainWindow(object):
./widgets/login/ui/login_dialog.py:class Ui_Dialog(object):
./program_turnon/ui/main_window.py:class Ui_main_window(object):
```

Reply:
I reverted to the names `read_hk_list` and `print_hk_list`, please run

    git pull origin

to update your local copy.